modules/developer-cli-odo-creating-a-java-microservice-jpa-application.adoc

```diff
@@ -0,0 +1,104 @@
+= Creating a Java MicroServices JPA application
+
+With `odo`, you can create and manage a sample Java MicroServices JPA application.
+
+.Procedure
+
+. Clone the sample application:
++
+[source,terminal]
+----
+$ git clone https://github.com/OpenLiberty/application-stack-samples.git
+----
+
+. Navigate to the application directory:
++
+[source,terminal]
+----
+$ cd ./application-stack-samples/jpa
+----
+
+. Initialize the project:
++
+[source,terminal]
+----
+$ odo create java-openliberty java-application
+----
+
+. Push the application to the cluster:
++
+[source,terminal]
+----
+$ odo push 
+----
++
+The application is now deployed to the cluster. 
+
+. See the status of the cluster by streaming the OpenShift logs to the terminal:
++
+[source,terminal]
+----
+$ odo log
+----
++
+Notice the failing tests because of an `UnknownDatabaseHostException` error since you application does not have a database yet:
++
+[source,terminal]
+----
+[INFO] [err] java.net.UnknownHostException: ${DATABASE_CLUSTERIP}
+[INFO] [err]    at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:220)
+[INFO] [err]    at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:403)
+[INFO] [err]    at java.base/java.net.Socket.connect(Socket.java:609)
+[INFO] [err]    at org.postgresql.core.PGStream.<init>(PGStream.java:68)
+[INFO] [err]    at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:144)
+[INFO] [err]    ... 86 more
+[ERROR] Tests run: 2, Failures: 1, Errors: 1, Skipped: 0, Time elapsed: 0.706 s <<< FAILURE! - in org.example.app.it.DatabaseIT
+[ERROR] testGetAllPeople  Time elapsed: 0.33 s  <<< FAILURE!
+org.opentest4j.AssertionFailedError: Expected at least 2 people to be registered, but there were only: [] ==> expected: <true> but was: <false>
+        at org.example.app.it.DatabaseIT.testGetAllPeople(DatabaseIT.java:57)
+
+[ERROR] testGetPerson  Time elapsed: 0.047 s  <<< ERROR!
+java.lang.NullPointerException
+        at org.example.app.it.DatabaseIT.testGetPerson(DatabaseIT.java:41)
+
+[INFO]
+[INFO] Results:
+[INFO]
+[ERROR] Failures:
+[ERROR]   DatabaseIT.testGetAllPeople:57 Expected at least 2 people to be registered, but there were only: [] ==> expected: <true> but was: <false>
+[ERROR] Errors:
+[ERROR]   DatabaseIT.testGetPerson:41 NullPointer
+[INFO]
+[ERROR] Tests run: 2, Failures: 1, Errors: 1, Skipped: 0
+[INFO]
+[ERROR] Integration tests failed: There are test failures.
+----
+
+. Create an ingress URL to access the application:
++
+[source,terminal]
+----
+$ odo url create --port 8080
+----
+
+. Push the changes:
++
+[source,terminal]
+----
+$ odo push
+----
+
+. Display the created URL:
```

Comment:
To verify, display the ingress URL that is created:

Reply:
Disagree here. You don't need this URL for verification only, you need it to access the application. 
"that is created" does not improve "the created URL" in any way, I don't see a reason to change the phrase.